examples/llama.py

```diff
@@ -514,9 +514,7 @@ def main(string: str):
   sys.stdout.write(outputted)
   sys.stdout.flush()
 
-  if args.profile:
-    import cProfile, pstats
-    profiler = cProfile.Profile()
+  if args.profile: from test.test_net_speed import start_profile, stop_profile
```

Comment:
You shouldn't import from test here, can abstract this functionality to `extra/helpers.py`